pkg/network/additional_networks_test.go

```diff
@@ -65,3 +115,71 @@ func TestValidateRaw(t *testing.T) {
 	rawConfig.Name = ""
 	errExpect("Additional Network Name cannot be nil")
 }
+
+func TestRenderSimpleMacvlanConfig(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	for _, cfg := range NetworkAttachmentConfigSimpleMacvlan.Spec.AdditionalNetworks {
+		objs, err := renderSimpleMacvlanConfig(&cfg, manifestDir)
+		g.Expect(err).NotTo(HaveOccurred())
+		g.Expect(objs).To(HaveLen(1))
+		g.Expect(objs).To(
```

Comment:
I'd like to see a test that validates the NetworkAttachmentDefinition content. You can use the "MatchYAML" and MatchJSON functions of gomega, if you like.

Reply:
added.